src/main/java/com/example/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,21 @@
 package com.example.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
     private Long id;
+    @NotNull
```

Comment:
I believe it`s better to use the `@NotBlank` constraint for a string in general and in this case especially.
**[baeldung](https://www.baeldung.com/java-bean-validation-not-null-empty-blank)**